charts/apiclarity/templates/deployment.yaml

```diff
@@ -27,6 +27,8 @@ spec:
             do echo waiting for database; sleep 2; done;']
           securityContext:
             runAsUser: 1001
+          resources:
+          {{- toYaml .Values.apiclarity.resources | nindent 13 }}
```

Comment:
Not sure that we need `resources` for the init container, and if so it probably need to have different values than the main container.

Reply:
The resources are necessary since it is possible to set a quota which will error if no requests/limits are specified. Fair point though, I'll make it a separate value.